README.md

```diff
@@ -256,6 +257,30 @@ Find groups which have linked with a feedId (see `ssb.tribes.link.create`).
 
 NOTE: the strange format with states is to leave easy support for multiple editors (of a link to a group) in the future
 
+### `ssb.tribes.subtribe.findByGroupId(groupId, cb)`
+
+Find subgroups which have linked with a groupId (see `ssb.tribes.link.createSubgroupLink`).
+
+- `groupId` *GroupId* is a string
+- `cb` *function* is a callback with signature `cb(err, data)` where `data` is an Array of items of form:
+  ```js
+  {
```

Comment:
maybe put this in a [] to lock in the Array idea

Reply:
good catch